packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
The only change in x-codemode package was in tsconfig.json https://github.com/mui/mui-x/pull/9068/files#diff-cb0344ec2c9ac329ecfec8b5e7b92c9788dd7b702efc6c2da08e59fa6ad8abe5
Seems safe to skip this package release. cc @flaviendelangle